process/process.go

```diff
@@ -133,6 +201,76 @@ func (p *process) initCapability() error {
 	return nil
 }
 
+// Camouflage() method's purpose is to adjust the process personality to another
+// process. The ultimate goal is to be able to utilize a 'proxy' process (such
+// as 'sysbox-fs nsenter') to execute instructions on behalf of the original
+// process.
+func (p *process) Camouflage(
```

Comment:
I would call it "SetPersonality"; "camouflage" sounds like it's hiding something. Just a suggestion.

Reply:
Not sure about this one, i think i like "camouflage" more than "setPersonality". Will think about it.

Reply:
Changed to "adjustPersonality".